app/src/main/res/layout/item_bill.xml

```diff
@@ -20,77 +21,99 @@
 
     </data>
 
-    <LinearLayout
+    <android.support.v7.widget.CardView
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:gravity="center"
-        android:orientation="horizontal"
-        android:padding="@dimen/dp_5">
+        android:layout_marginBottom="@dimen/dp_8"
+        car_view:cardElevation="@dimen/dp1"
+        >
 
         <LinearLayout
-            android:layout_width="0dp"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginEnd="@dimen/dp_10"
-            android:layout_marginRight="@dimen/dp_10"
-            android:layout_weight="1"
-            android:background="@color/color_white"
+            android:gravity="center"
             android:orientation="horizontal"
-            android:padding="@dimen/dp_10">
+            >
 
             <TextView
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:layout_marginEnd="@dimen/dp_30"
-                android:layout_marginRight="@dimen/dp_30"
-                android:text="@{item.serviceName}"
-                android:textColor="@color/color_black"
-                android:textSize="@dimen/sp_10"/>
+                android:background="@color/colorPrimary"
+                android:padding="@dimen/dp_2"
+                android:text="@{position + 1 + ``}"
+                android:textColor="@color/color_white"
+                android:textSize="@dimen/sp_18"/>
 
-            <TextView
-                android:layout_width="wrap_content"
+            <LinearLayout
+                android:layout_width="0dp"
                 android:layout_height="wrap_content"
-                android:layout_marginEnd="@dimen/dp_30"
-                android:layout_marginRight="@dimen/dp_30"
-                android:text="@{item.stylistName}"
-                android:textColor="@color/color_black"
-                android:textSize="@dimen/sp_10"/>
+                android:layout_marginEnd="@dimen/dp_10"
+                android:layout_marginRight="@dimen/dp_10"
+                android:layout_weight="1"
+                android:background="@color/color_white"
+                android:orientation="horizontal"
+                android:paddingLeft="@dimen/dp_5">
```

Comment:
WARNING: [RtlSymmetry] When you define `paddingLeft` you should probably also define `paddingRight` for right-to-left symmetry
 If you specify padding or margin on the left side of a layout, you should probably also specify padding on the right side (and vice versa) for right-to-left layout symmetry.